src_assets/common/assets/web/public/assets/locale/en.json

```
@@ -365,9 +367,13 @@
     "restart_sunshine_success": "Sunshine is restarting",
     "troubleshooting": "Troubleshooting",
     "unpair_all": "Unpair All",
-    "unpair_all_desc": "Remove all your paired devices",
+    "unpair_all_desc": "Remove your paired devices. Individually unpaired devices with an active session will remain connected, but cannot start or resume a session.",
```

```suggestion
    "unpair_desc": "Remove your paired devices. Individually unpaired devices with an active session will remain connected, but cannot start or resume a session.",
```

The key was throwing me off. If this is the generic description now for all of the unpairing options we should rename the key.

---

Updated 👍 https://github.com/LizardByte/Sunshine/pull/2042/commits/52cb99611012dabdebb8caae51fb211bd8fdc58b